packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -761,80 +757,202 @@ export class InputTimePicker
   }
 
   private getExtendedLocaleConfig(
-    locale: string,
+    locale: SupportedLocale,
   ): Parameters<(typeof dayjs)["updateLocale"]>[1] | undefined {
     if (locale === "ar") {
```

Comment:
WDYT about adding a general comment at the top as a reference for each locale? For example:

```tsx
/*
 * Meridiem and format tokens below are based on https://github.com/unicode-org/cldr-json/
 * 
 * To reference a specific locale, check:
 * https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-dates-modern/main/<locale>/ca-generic.json
 * 
 * Example (es-MX):
 * https://github.com/unicode-org/cldr-json/blob/d38478855dd8342749f0494332cc8acc2895d20d/cldr-json/cldr-dates-modern/main/es-MX/ca-generic.json#L227
 */
```

It provides enough context, helps avoid outdated comments, and reduces boilerplate when editing custom locale configs.